2021-04-25 17:19:43,272 INFO    MainThread:52502 [wandb_setup.py:_flush():69] setting env: {}
2021-04-25 17:19:43,272 INFO    MainThread:52502 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-25 17:19:43,273 INFO    MainThread:52502 [wandb_init.py:_log_setup():336] Logging user logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_171943-2p4dag76/logs/debug.log
2021-04-25 17:19:43,273 INFO    MainThread:52502 [wandb_init.py:_log_setup():337] Logging internal logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_171943-2p4dag76/logs/debug-internal.log
2021-04-25 17:19:43,273 INFO    MainThread:52502 [wandb_init.py:init():369] calling init triggers
2021-04-25 17:19:43,274 INFO    MainThread:52502 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-25 17:19:43,274 INFO    MainThread:52502 [wandb_init.py:init():418] starting backend
2021-04-25 17:19:43,274 INFO    MainThread:52502 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-25 17:19:43,293 INFO    MainThread:52502 [backend.py:ensure_launched():123] starting backend process...
2021-04-25 17:19:43,302 INFO    MainThread:52502 [backend.py:ensure_launched():128] started backend process with pid: 52518
2021-04-25 17:19:43,304 INFO    MainThread:52502 [wandb_init.py:init():423] backend started and connected
2021-04-25 17:19:43,307 INFO    MainThread:52502 [wandb_init.py:init():463] updated telemetry
2021-04-25 17:19:43,307 INFO    MainThread:52502 [wandb_init.py:init():482] communicating current version
2021-04-25 17:19:44,023 INFO    MainThread:52502 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-25 17:19:44,023 INFO    MainThread:52502 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-25 17:19:44,202 INFO    MainThread:52502 [wandb_init.py:init():520] starting run threads in backend
2021-04-25 17:19:44,266 INFO    MainThread:52502 [wandb_run.py:_console_start():1524] atexit reg
2021-04-25 17:19:44,267 INFO    MainThread:52502 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-25 17:19:44,268 INFO    MainThread:52502 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-25 17:19:44,271 INFO    MainThread:52502 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-25 17:19:44,271 INFO    MainThread:52502 [wandb_init.py:init():544] run started, returning control to user process
2021-04-25 17:19:53,974 INFO    MainThread:52502 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-25 17:19:53,974 INFO    MainThread:52502 [wandb_run.py:_restore():1466] restore
2021-04-25 17:19:56,143 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 17:19:56,511 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 17:19:57,228 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 1949
}

2021-04-25 17:19:57,418 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 1966
}

2021-04-25 17:19:57,521 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:19:57,623 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:19:57,725 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:19:57,828 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:19:57,931 INFO    MainThread:52502 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:19:59,094 INFO    MainThread:52502 [wandb_run.py:_show_files():1838] logging synced files
